src/index.js

```diff
@@ -1844,5 +1844,21 @@ module.exports = {
         }
       }
     );
+
+    expect.exportAssertion(
+      '<DOMElement> [not] to have focus',
+      (expect, subject) => {
+        expect(subject.ownerDocument.activeElement, '[not] to be', subject);
```

Comment:
Don't we need a similar guard here for when dereferencing `activeElement` throws an error? It seems fine to assume that the element does not have focus when that happens.

```suggestion
        let hasFocus = false;
        try {
          hasFocus = subject === subject.ownerDocument.activeElement;
        } catch (err) {}
        expect(hasFocus, '[not] to be true');
```

(would be good to have a test covering this as well)

Reply:
I wasn't sure how to recreate the problem in a test.

Looks like I also reversed subject and comparator in this assertion.
Applying the change. I assume the failure output stays the same because of the unchanged errorMode